README.md

```diff
@@ -157,6 +157,21 @@ All columns are sortable by default, but sorting can be disabled by setting `sor
     { key: 'year', label: 'Year', sortable: false }
 
 
+##### Table Header Column CSS Class
+
+To set the css class for table header **&lt;th&gt;**, use the optional *thclass* key. This attribute can be a String or a Function.
```

Comment:
That list is for top-level settings and this is a field setting, so I think it's fine where it is.